### ISSUE TITLE
No authentication handler is configured to authenticate for the scheme: Token

### ISSUE THREAD

**Issue by jackjwilliams** [gothinkster/aspnetcore-realworld-example-app#9]
Cloned this project and the React / Redux client project, ran both but when posting an article it gives the error message:

No authentication handler is configured to authenticate for the scheme: Token

**Comment by adamhathcock** [comment]
Looks to be because that client uses `Token` instead of `Bearer` which is related to this: https://github.com/gothinkster/realworld/issues/81

**Comment by jackjwilliams** [comment]
Any proposed fix on either the client or server side? I've tried changing the React client to use Bearer instead of token, removing the JwtBearer OnMessageReceived event server side, but it always returns the same error.

I can register fine, but any attempt to post an article, update my settings, or anything - results in this error.

Thanks for the repo btw! It's clean and simple.

**Comment by adamhathcock** [comment]
This is weird.  I knew I did something: https://github.com/gothinkster/aspnetcore-realworld-example-app/blob/master/src/Conduit/StartupExtensions.cs#L65

I take the `Authorization` header and look for `Bearer` then change it to `Token` but if it's already `Token` then it should be fine.

Can you look at the headers actually being sent in the browser dev tools or try with Postman?  I'd like to see a screenshot or something of what's being sent.

**Comment by jackjwilliams** [comment]
I noticed that, which is why I was baffled by this error. I finally just changed the scheme to Bearer everywhere and that got me past those errors.

But then I found others. I would recommend testing your backend with a few of the front ends (React, for example). A clean checkout of react real world pointed to this backend will yield other errors and oddities (errors don't show up on failed registration because RestException is caught and returned with only a status code, viewing an article doesn't work due to missing username / authorname on comments json).

If I get a chance I'll throw up a PR for some of the fixes I did.
